### PR TITLE
a few updates

### PR DESCRIPTION
**Pull request by Xxyuri2005xX**
Remove Deprecated from Messages class
Implement halloween disable feature

### CODE REVIEW

File: bedwars-api/src/main/java/com/andrei1058/bedwars/api/language/Messages.java [hunk 1]
```diff
@@ -23,7 +23,6 @@
 import com.andrei1058.bedwars.api.configuration.ConfigPath;
 
 @SuppressWarnings("WeakerAccess")
-@Deprecated
```
Comment (tomkeuper) [inline bedwars-api/src/main/java/com/andrei1058/bedwars/api/language/Messages.java:26]: You cant just remove the Deprecated warning...

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/configuration/MainConfig.java [hunk 2]
```diff
@@ -51,6 +51,7 @@ public MainConfig(Plugin plugin, String name) {
         yml.addDefault(ConfigPath.GENERAL_CONFIGURATION_DISABLED_LANGUAGES, Collections.singletonList("your language iso here"));
         yml.addDefault("storeLink", "https://www.spigotmc.org/resources/authors/39904/");
         yml.addDefault("lobbyServer", "hub");
+        yml.addDefault(ConfigPath.GENERAL_CONFIGURATION_ENABLE_HALLOWEEN, true);
```
Comment (tomkeuper) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/configuration/MainConfig.java:54]: Default halloween on?

**Comment by tomkeuper** [comment]
I am not sure what this pr is about. You have lots of commits but ended up only changing some variables and added one config option

File: bedwars-api/src/main/java/com/andrei1058/bedwars/api/language/Messages.java (hunk 1 above)
Comment (Xxyuri2005xX) [reply bedwars-api/src/main/java/com/andrei1058/bedwars/api/language/Messages.java:26]: I removed only the deprecated that was being given to the whole class without interfering with the messages that I would go has deprecated with the sidebar update

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/configuration/MainConfig.java (hunk 2 above)
Comment (Xxyuri2005xX) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/configuration/MainConfig.java:54]: I have defaulted the function to true but in the config anyone who wants can disable it.
If yuou want i can set that on false as default.

**Comment by Xxyuri2005xX** [comment]
some commits come from a small error caused by me, they are commits that can be ignored as they are already merged .

**Review by andrei1058 — APPROVED** [review]

**Pull request merged**